src/main/java/dev/openfga/sdk/api/client/OpenFgaClient.java

```diff
@@ -273,19 +277,20 @@ public CompletableFuture<ClientWriteResponse> write(ClientWriteRequest request,
         configuration.assertValid();
         String storeId = configuration.getStoreIdChecked();
 
-        WriteRequest body = new WriteRequest();
+        if (options != null && (options.disableTransactions() || options.getTransactionChunkSize() == 1)) {
```

Comment:
curious why we want to treat chunksize = 1 differently?

Reply:
I got the idea from comments I read in some of the SDKs. I'm not sure if I read the comment correctly, or if behavior changes in the 1 case:

* https://github.com/openfga/dotnet-sdk/blob/be2d963baae1e38c0f7ec21323937c26b2b80954/src/OpenFga.Sdk/Client/Client.cs#L166
* https://github.com/openfga/js-sdk/blob/78eb4cfdea14455f0b63d3f5f98c78f8209a09ce/client.ts#L369

Talked offline and we will not do any special behavior when chunk size is 1 here